### ISSUE TITLE
NTR: Dopamine beta-hydroxylase activity undetectable in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8224]
New term request Dopamine beta-hydroxylase activity undetectable in blood (elthat_130220203056):   UMLS: 1857218

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000556
- label: Reduced circulating dopamine beta-hydroxylase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:11857564
- def: Concentration of Dopamine beta-hydroxylase (DBH; EC 1.14.17.1) in the blood circulation below the lower limit of normal. DBH catalyzes the oxidative hydroxylation of dopamine to norepinephrine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8224
- synonyms:
